src/Tribe/Embedded_Maps.php

```diff
@@ -94,14 +94,27 @@ public function get_map( $post_id, $width, $height, $force_load ) {
 		end( $this->embedded_maps );
 		$index = key( $this->embedded_maps );
 
-		// Generate the HTML used to "house" the map
 		ob_start();
 
-		tribe_get_template_part( 'modules/map', null, array(
-			'index'  => $index,
-			'width'  => null === $width  ? apply_filters( 'tribe_events_single_map_default_width', '100%' ) : $width,
-			'height' => null === $height ? apply_filters( 'tribe_events_single_map_default_height', '350px' ) : $height,
-		) );
+		if ( tribe_is_using_basic_gmaps_api() ) {
+
+			// Get a basic embed that doesn't use the JavaScript API
+			tribe_get_template_part( 'modules/map-basic', null, array(
+				'embed_url'   => tribe_get_basic_gmap_embed_url( $this->address ),
```

Comment:
Maybe use the `if...else` to fill in the args array and then make a call to the template part? (It'll be done anyway, but with different args)

Reply:
Not sure what you mean here—I _think_ you're saying to try and consolidate this into just one `tribe_get_template_part()` call, with the if/else block just modifying the args passed to the call.

I could see the value of that, but since these are two different templates I think it's more clear to have two different `tribe_get_template_part()` calls, and for the sake of time will leave this for now.

Reply:
Yes, that's what I meant. And my bad, I didn't noticed that. Thanks for the heads up